playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -97,6 +97,10 @@ EDXAPP_FORUM_MONGO_PARAMS:
       port: "{{ FORUM_MONGO_PORT|default('27017') }}"
   password: "{{ FORUM_MONGO_PASSWORD|default('password') }}"
   user: "{{ FORUM_MONGO_USER|default('cs_comments_service') }}"
+EDXAPP_MARKETO_API_URL: ''
+EDXAPP_MARKETO_CLIENT_ID: ''
+EDXAPP_MARKETO_CLIENT_SECRET: ''
+EDXAPP_MARKETO_LIST_ID: 0
```

Comment:
`null` might be better so it's clear it was not provided, it's also what they use [here](https://github.com/Stanford-Online/configuration/blob/675280ad71ed7f56d0f90c5e4b4022efc6a4974a/playbooks/roles/edxapp/defaults/main.yml#L772)

Reply:
@kluo I went with `0` here to make it obvious that this value _must_ be an integer.

You can _not_ use:
- `EDXAPP_MARKETO_LIST_ID: "1234"`

you must use:
- `EDXAPP_MARKETO_LIST_ID: 1234`

From the code (here and platform), I can't tell if the same applies to the mailchimp ID.

Thoughts?

Reply:
Okay 👍